Frontend/ui-common/src/Engineering/Helpers/LogEvent.purs

```diff
@@ -0,0 +1,88 @@
+module Engineering.Helpers.LogEvent
+  where
+
+import Effect (Effect, foreachE)
+import Data.Maybe(Maybe(..), fromMaybe, isJust)
+import Data.Array (find)
+import Data.Either (Either(..))
+import Tracker.Labels (Label(..))
+import Foreign (Foreign, readString)
+import Control.Monad.Except (runExcept)
+import Foreign.Class (class Encode, encode)
+import Foreign.Object (Object, empty, insert, lookup)
+import Tracker (trackActionObject, trackScreenEnd, trackScreen, trackScreenEvent)
+import Tracker.Types (Action(..), Level(..), Screen(..)) as Tracker
+import Prelude (Unit, pure, unit, ($), (<$>), (<<<), (/=), (==), (&&), bind, void, when, discard,map)
+import Presto.Core.Types.Language.Flow (Flow , doAff)
+import JBridge (firebaseLogEvent, firebaseLogEventWithParams, firebaseLogEventWithTwoParams)
+import Log (rootLevelKeyWithRefId)
+import Debug (spy)
+import Effect.Class (liftEffect)
+
+foreign import getLogDestination :: Effect (Array String)
+
+isElement :: String -> Array String -> Boolean
+isElement logBrand logBrands = isJust $ find (\el -> el == logBrand) logBrands
+
+triggerLog :: String -> Object Foreign -> String -> Effect Unit 
+triggerLog event logField logDestination = case logDestination of  
+  "JUSPAY" -> do 
+    _ <- pure $ spy "second one" " "
+    _ <- firebaseLogEvent event 
+    pure unit 
+  "FIREBASE" -> do 
+    let 
+      eventObject = insert "event" (encode event) empty
+      foreignObject = rootLevelKeyWithRefId logField
+    _ <- trackActionObject Tracker.User Tracker.Info ON_EVENT eventObject foreignObject
+    pure unit 
+  _ -> do 
+    _ <- pure $ spy  "heghfvj" ""   
```

Comment:
@SanskarGubreley don't commit log , If require add proper log

Reply:
will remember it.